report/src/system_evaluation/results.tex

```diff
@@ -32,28 +32,40 @@ \section{System evaluation}\label{sec:experiment_results}
+    \label{lst:resultsTablet}
+\end{table}
 
-To create an environment that more accurately represents a meeting room in use, several devices were used for testing. 
-The method for collecting results from these tests mirrored previous ones, by making $10$ classifications on both a Samsung Galaxy A53, a Samsung Galaxy S20 phone, and a Lenovo TabM10 FHD Plus tablet. 
-The classification results were gathered in the locations shown on Figure \ref{fig:experiment_room}
-During the experiment, k-values of $2$, $3$, and $4$ were used.  
-The results of the experiment can be seen in Tables \ref{lst:resultsPhone}, \ref{lst:resultsPhoneS20}, and  \ref{lst:resultsTablet}.
-Table \ref{lst:resultsPhone_precision} summarizes the experiment's results, indicating the percentage of true positive classifications. 
+On Table \ref{lst:resultsPhone_precision} see that a lower $k$-value results in a larger uncertainty in the classification.
+However, it is worth noticing that for measuring points \textit{B} and \textit{J}  (see Figure \ref{fig:experiment_setup}) a large uncertainty in classification is observed.
+We suspect that this is a consequence of the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it. 
+This low absorbation results in the two data points next to the pane having similar distances to both map values labelled as inside and some as outside, therefore giving uncertain classifications.
+
+Similarly, for all devices, when performing the classification in locations H, I, J, K, and L shows, the results are vastly different from the other locations.
+These locations are all outside of the room. 
+We suspect that this can be due to two things.
+First and foremost, it might be due to the created map being unreliable. 
+When creating the map, the wall separating the room and the common area might cause large fluctuations in the strength of the received signal. 
+Therefore, classification using the map becomes unreliable.
+We tried to accomodate this by enforcing all map entries to have a standard deviation of 6.5 \todo{units and 6.5}. 
+Enforcing a lower standard deviation of the map might alleviate these issues.
+However, this will increase the time it takes to create the map, as more samples will be required if the signal fluctuates.
+Furthermore, detecting these signal outliers might be difficult if the signal often fluctuates by high degrees.      
+
+Secondly, by having placed beacons only inside of the meeting room, and by having more classification data representing locations inside the meeting room, the classification results might be skewed towards \textit{inside} classifications. 
+One could try lessen the effect of this skewing by placing one or more beacons in the common area, and treat signals from this beacon as a weight towards a classification of \textit{outside}.  
```

Comment:
The weight thing: that's not how it works.
Also, explain why it might help.

```suggestion
One could try to lessen the effect of this bias by placing one or more beacons in the common area.
```

Reply:
We need to explain why this resolves the bias.

Reply:
Not sure why we think it would.

Trying to steelman:
- Fluctuations are due to beacons being behind the wall, and therefore placing beacon(s) outside the room helps stabilize signal strengths

But the walls & their material are constants. Would it really make a difference? Why wouldn't it just be consistently bad signal, and therefore make no difference?

I think an effort to limit confounding variables, e.g. by keeping as many variables constant as we can, would show an improvement. Perhaps also placing an equivalent amount of grid points inside & outside?